jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/model/CodeModernizerSessionContext.kt

```diff
@@ -225,6 +244,84 @@ data class CodeModernizerSessionContext(
             }
         }
 
+        // 3. intellij-bundled maven runner
+        if (shouldTryMvnCommand) {
+            LOG.warn { "Executing IntelliJ bundled Maven" }
+            val goalcp = "dependency:copy-dependencies"
+            val outputDirectory = "-DoutputDirectory=$destinationDir"
+            val repolay = "-Dmdep.useRepositoryLayout=true"
+            val pomcp = "-Dmdep.copyPom=true"
+            val parentpom = "-Dmdep.addParentPoms=true"
+            val commandlist = mutableListOf<String>()
+            val explicitenabled = mutableListOf<String>()
+            commandlist.add(goalcp)
+            commandlist.add(outputDirectory)
+            commandlist.add(repolay)
+            commandlist.add(pomcp)
+            commandlist.add(parentpom)
```

Comment:
a bit too verbose, just build the list inline
```suggestion
            val explicitenabled = emptyList<String>()
            val commandlist = buildList {
                add("dependency:copy-dependencies")
                add("-DoutputDirectory=$destinationDir")
                add("-Dmdep.useRepositoryLayout=true")
                add("-Dmdep.copyPom=true")
                add("-Dmdep.addParentPoms=true")
            }
```

Reply:
yes. the command list is same as `runMavenCommand()` .

Reply:
should we dedupe?

Reply:
Deduped. Thanks.